python-threatexchange/version.txt

```diff
@@ -1 +1 @@
-1.2.1
```

Comment:
Rewinding this will probably fail the test publish, since I already put it there, but will retry assuming build succeeds